src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md

```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
 
-## Sharing the dashboard
+The primary visualizations on these pages are the monthly distributions of
+experiences categorized as "Good", "Needs Improvement", and "Poor". The 
+color-coded legend below the chart indicates the range of experiences included
+in the category. For example, in the screenshot above, you can see the 
+percent of good [Largest Contentful Paint](/lcp/) (LCP) experiences, which are 
+categorized as those less than 2500ms, fluctuating and getting slightly worse 
+in recent months.
 
-Data Studio dashboards can be shared just like a Google Doc to provide access to
-your team or to make the dashboard publicly viewable by permalink. Needless to
-say, you don't have to own the site you are getting a dashboard for, so you can 
-easily use this tool to keep an eye on your competitors' sites.
+The most recent month's percentages of good and poor experiences are shown 
+above the chart along with an indicator of the percent difference from the 
+previous month. For this origin, good LCP experiences fell by 3.2% to 56.04% 
+month-over-month.
 
-![CrUX Data Studio embedding and sharing](./dash_embed.png)
+{% Aside 'caution' %}
+Due to a quirk with Data Studio, you may sometimes see "No Data" here. This is
+normal and due to the previous month's release not being available until the
+second Tuesday.
+{% endAside %}
 
-Dashboards may also be embedded into websites or blog posts and retain all of
-their interactivity features.
+Additionally, for metrics like LCP and other Core Web Vitals that provide 
+explicit percentile recommendations, you'll find the "P75" metric between the 
+good and poor percentages. This value corresponds to the origin's 75th 
+percentile of user experiences. In other words, 75% of experiences are better
+than this value. One thing to note is that this applies to the overall 
+distribution across _all devices_ on the origin. Toggling specific devices 
+with the "Device" filter will not recalculate the percentile.
+
+{% Details %}
+{% DetailsSummary %}
+Boring technical caveats about percentiles
+{% endDetailsSummary%}
+
+Be aware that the percentile metrics are based on the histogram data from 
+[BigQuery](/chrome-ux-report-bigquery/), so the granularity will be coarse: 
+1000ms for LCP, 100ms for FID, and 0.05 for CLS. In other words, a P75 LCP of
+3800ms indicates that the true 75th percentile is somewhere between 3800ms and
+3900ms.
+
+Additionally, the BigQuery dataset uses a technique called "bin spreading" in 
+which densities of user experiences are intrinsically grouped into very coarse 
+bins of decreasing granularity. This allows us to include minute densities in 
+the tail of the distribution without having to exceed four digits of 
+precision. For example, LCP values less than 3 seconds are grouped into bins 
+200ms wide. Between 3 and 10 seconds, bins are 500ms wide, then 5000ms wide, 
+etc. Rather than having bins of varying widths, bin spreading ensures that all 
```

Comment:
```suggestion
200ms wide. Between 3 and 10 seconds, bins are 500ms wide. Beyond 10 seconds, bins are 5000ms wide, 
etc. Rather than having bins of varying widths, bin spreading ensures that all 
```
I stumbled over this paragraph. If my understanding is correct then I think this phrasing will make it easier to grok on first try